chapters/3-measurement.tex

```diff
@@ -160,30 +208,22 @@ \subsection{Master datasets}
 Examples of such activities are data collection, data analysis,
 random sampling, and treatment assignment.\index{master dataset}
 The master dataset should be the authoritative source of
-all \textbf{research variables}\sidenote{
-	\textbf{Research variables:} Research data that identifies observations
-	and maps research design information to those observations.
-	Research variables are time-invariant and
-	often, but not always, controlled by the research team.
-	Examples include
-	ID variables, sampling status, treatment status, and treatment uptake.}\index{research variables}
-but not include any \textbf{measurement variables}.\sidenote{
-	\textbf{Measurement variables:} Data that
-	measures attributes or records responses of research subjects.
-	Measurement variables are not controlled by the research team
-	and often vary over time.
-	Examples include characteristics of the research subject,
-	outcome variables, and control variables among many others.}\index{measurement variables}
-Research variables and measurement variables
+all research design variables\index{research design variables}
+but not include any measurement variables.
+Research design variables and measurement variables
 often come from the same source,
 but should not be stored in the same way.
 For example, if you acquire administrative data that both includes
-information on eligibility for the study (research variable)
-and data on the topic of your study (measurement variable)
-you should first decide which variables are research variables,
+information on eligibility for the study (research design variables)
+and data on the topic of your study (measurement variables)
+you should first decide which variables are research design variables,
 and store them in the master dataset,
 while storing the measurement variables in a cleaned dataset
 as described in Chapter 5.
+This distinction is made as variables that 
+describes your research design applies across your project,
+but measurement variables is a snapshot 
+that describe the point of time they were observed in.
```

Comment:
David commented that " I also found it odd to think of treatment take-up as part of this – sometimes treatment take-up is time-varying". That's a fair point. I don't think we necessarily want to say that design variables are time invariant, nor that measurement variables are a snapshot; they can measure time-invariant characteristics. I think the distinctions we drew earlier between the two types of variables are clearer and sufficient. I've removed this in my edits but we can discuss.